File: tests/.java
import java.util.*;

public class Java {
    public static void main(String[] args) {
        HashMap<String, String> map = new HashMap<String, String>();

        System.out.println("Hello, world!");

        for (int i = 0; i < 10; i++) {
            map.put("key" + i, "value" + i);
        }
    }
}

File: tests/.cs
using System.Collections.Generic;

public class MyProgram {
    public static void main(String[] args) {
        int answer = 42;
        
        for (int i = 0; i < 10; i++) {
            Console.WriteLine(i);
        }

        HashSet<double> set = new HashSet<double>();

        set.Add(69420d);

        Console.WriteLine("Hello, world!");
    }
}